src/App.js

```diff
@@ -0,0 +1,71 @@
+/*
+We anticipate rendering the following here:
+NavigationBar
+LeafletMap
+*/
+import React, {useDebugValue, useState} from 'react';
+import { NavigationBar } from './components/NavigationBar';
+import { LeafletMap } from './components/LeafletMap';
+import ReactDOM from 'react-dom';
+import {makeStyles} from "@material-ui/core/styles";
+import config from "./config";
+import {getNextFWDate, toWMSDate} from "./datemanagement";
+import L from "leaflet";
+import { useStateWithLabel, getWMSLayersYearRange } from "./utils";
+
+
+export default function App() {
+
+    // Initialize Material UI styles
+    const useStyles = makeStyles({
```

Comment:
CSS classes should be in each individual component and not here in App. Any common CSS that we notice across the components can be abstracted as a theme.

Reply:
ThemeProvider, createTheme both from MUI - see crest v2 theme for example on how to bring it in: 

See how createTheme is used in CRESTCustomTheme and ThemeProvider is used in App.js
https://github.com/nemac/crest-v2/blob/master/src/CRESTTheme/CRESTCustomTheme.js
https://github.com/nemac/crest-v2/blob/master/src/App.js